src/Definition/Parameter.php

```diff
@@ -1,82 +1,73 @@
-<?php
+<?php declare(strict_types=1);
+
 namespace ElevenLabs\Api\Definition;
 
+use stdClass;
```

Comment:
The `object` type is only available in PHP ^7.2, so we are limited to `stdClass` hints, which I really do not like.

Thoughts on making the minimum requirement `^7.2` instead? I realize it might turn a few people away initially but PHP 7.3 is about to be released...